portfolio/src/main/webapp/owl.carousel/CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+## Contributing
```

Comment:
You are using OWL unmodified right? So you should use the "distributed" version per description here https://owlcarousel2.github.io/OwlCarousel2/docs/started-welcome.html?

Reply:
gotcha it has been replaced!